Sources/RunnerLib/SPMDanger.swift

```diff
@@ -28,7 +30,15 @@ public struct SPMDanger {
         _ = try? executor.shellOut(command: "swift build --product \(depsLibName)")
     }
 
-    public var libImport: String {
-        return "-l\(depsLibName)"
+    public var libsImports: [String] {
+        let libsImport = ["-l\(depsLibName)"]
+
+        // The danger lib is not always generated, this mainly happens on the danger repo, where the DangerDeps library and Danger.swiftmodule are enough
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/SPMDanger.swift;
  Line: 36;
-->

- :warning: Line should be 120 characters or less: currently 153 characters (`line_length`)